src/index.js

```diff
@@ -18,11 +18,10 @@ module.exports = (
     omitOptions = [],
   } = {}
 ) => {
-  bookshelf.plugin('visibility');
   if (caseConverter) {
-    bookshelf.plugin('case-converter');
+    bookshelf.plugin('bookshelf-case-converter-plugin');
```

Comment:
https://github.com/bookshelf/bookshelf/wiki/Migrating-from-0.15.1-to-1.0.0#case-converter-plugin